pyaedt/icepak.py

```diff
@@ -2609,11 +2680,11 @@ def apply_icepak_settings(
             Whether to perform minimal validation. The default is ``True``.
             If ``False``, full validation is performend.
         default_fluid : str, optional
-            Default for the type of fluid. The default is ``"Air"``.
+            Default type of fluid. The default is ``"Air"``.
         default_solid :
-            Default for  the type of solid. The default is ``"Al-Extruded"``.
+            Default type of solid. The default is ``"Al-Extruded"``.
         default_surface :
-            Default for the type of surface. The default is ``"Steel-oxidised-surface"``.
+            Default type of surface. The default is ``"Steel-oxidised-surface"``.
```

Comment:
```suggestion
            Type of surface. The default is ``"Steel-oxidised-surface"``.
```
The U.S. English spelling is oxidized. The Ansys doc standards say to use U.S. English spellings.